jobsdb/jobsdb.go

```diff
@@ -2779,6 +2782,9 @@ func (jd *HandleT) backupDSLoop(ctx context.Context) {
 	for {
 		select {
 		case <-time.After(sleepMultiplier * backupCheckSleepDuration):
+			if !jd.BackupSettings.BackupEnabled {
+				return
```

Comment:
We are now making it possible to turn backup off without having to restart the server. Turning it back on is not something that we are interested in?

Reply:
That was a limitation, now this limitation is removed and we can turn that back up.